frontend/ui/data/css_tweaks.lua

```diff
@@ -161,7 +161,48 @@ h1 + h6, h2 + h6, h3 + h6, h4 + h6, h5 + h6 { page-break-before: avoid !importan
                 id = "headings_align_center",
                 title = _("Center headings"),
                 css = [[h1, h2, h3, h4, h5, h6 { text-align: center !important; }]],
-                priority = 3, -- so it overrides the ones above
+                priority = 4, -- so it overrides the ones above
+            },
+        },
+        {
+            title = _("Text direction"),
+            {
+                title = _("About text direction"),
+                info_text = _([[
+Documents in languages like arabic or hebrew are to be rendered Right-To-Left. This does not only affects text layout, but also list items bullets and numbers (put on the right) and table layout (cells laid from right to left).
```

Comment:
Maybe something like this? (In any case, Arabic and Hebrew are capitalized in English. ;-))
```suggestion
Languages like Arabic or Hebrew use right-to-left writing systems (Right-To-Left or RTL). This doesn't only affect text layout, but also list item bullets and numbers, which have to be put on the right side of the page, as well as and tables, where the cells are laid out from right to left.
```